10x/hca-dcp/wrapper_10x_count.wdl

```diff
@@ -183,7 +183,7 @@ task inputs_for_submit {
     CODE
   >>>
   runtime {
-    docker: "humancellatlas/secondary-analysis-python:0.1.4"
+    docker: "humancellatlas/pipeline-tools:0.1.4"
```

Comment:
I'd like to also re-tag secondary-analysis-python from 0.1.0 to 0.1.3 images to pipeline-tools and push them to the new docker repo, and remove secondary-analysis-python docker repo then we would have a continuous version changes.